concurrent_test.go

```diff
@@ -24,6 +24,22 @@ import (
 	"go.etcd.io/bbolt/internal/common"
 )
 
+type operationInterval struct {
+	minInterval time.Duration
+	maxInterval time.Duration
+}
+
+type writeBytesRange struct {
+	minWriteBytes int
+	maxWriteBytes int
+}
```

Comment:
```suggestion
type range struct {
	min int
	max int
}
```

Reply:
done

Reply:
Let's use `bytesRange` instead of `range`:
1. explicitly mention it's related to bytes;
2. not to conflict with the key word `range`